src/components/AssetCard/AssetCard.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useMemo } from 'react'
+import { Network, Rarity, Item } from '@dcl/schemas'
+import { WearablePreview } from '../WearablePreview'
+import { Mana } from '../Mana/Mana'
+import { Card } from '../Card/Card'
+import { Profile } from '../Profile/Profile'
+import { formatWeiToAssetCard, getCatalogCardInformation } from './utils'
+import './AssetCard.css'
+
+export type AssetCardTranslations = {
+  also_minting: string
+  available_for_mint: string
+  available_listings_in_range: string
+  cheapest_listing: string
+  cheapest_option_range: string
+  cheapest_option: string
+  listing: string
+  listings: string
+  most_expensive_range: string
+  most_expensive: string
+  not_for_sale: string
+  owners: string
+  owner: string
+}
+
+export type AssetCardFilters = {
+  price?: number
+  sortBy?: string
+  minPrice?: string
+  maxPrice?: string
+}
+
+export type AssetCardProps = {
+  translations: AssetCardTranslations
+  asset: Item
+  assetFilters: AssetCardFilters
+  onClickCardURL: string
+}
+
+export const AssetCard = (props: AssetCardProps) => {
+  const { asset, translations, onClickCardURL, assetFilters } = props
+
+  const catalogItemInformation = useMemo(() => {
+    return getCatalogCardInformation(asset, translations, assetFilters)
+    return null
+  }, [asset])
```

Comment:
This null return is redundant, would you mind removing it?
```suggestion
  const catalogItemInformation = useMemo(() => getCatalogCardInformation(asset, translations, assetFilters), [asset])
```